code/game/objects/auras/regenerating_aura.dm

```diff
@@ -119,13 +119,15 @@
 	grow_threshold = 150
 	ignore_tag = BP_HEAD
 	var/toggle_blocked_until = 0 // A time
+	var/cold_threshold = 280 //Under this threshold we cannot use healing
 
 /obj/aura/regenerating/human/unathi/toggle()
 	..()
 	toggle_blocked_until = max(world.time + 2 MINUTES, toggle_blocked_until)
 
 /obj/aura/regenerating/human/unathi/can_toggle()
-	if(world.time < toggle_blocked_until)
+	var/mob/living/carbon/human/H = user
+	if(world.time < toggle_blocked_until || H.bodytemperature < cold_threshold)
```

Comment:
Might want to add brackets for each case, unless you are 100% sure it works like that.
Byond is sometimes very weird there.

Reply:
I am 100% sure, otherwise the equality operators wouldn't function correctly.